viewer/vueapp/src/components/spigraph/Spigraph.vue

```diff
@@ -511,8 +511,9 @@ export default {
         respondedAt = Date.now();
         this.error = '';
         this.loading = false;
-        this.items = []; // clear items
-        this.processData(response.data);
+        this.items = response.data.items;
+        this.mapData = response.data.map;
```

Comment:
Can we add these inits to Sessions .vue

Reply:
I'm not sure what you mean. Can you clarify?

Reply:
The initializing for mapData and graphData. They were set to undefined like this line previously.

Reply:
Where were they set to undefined? 

So in Sessions.vue the mapData and graphData are set the same way as in Spigraph.vue. I took out the processData step because it is not needed and I wanted to simplify the code. Before, the processData function was used to set the mapData and graphData. But now it's all done here, like in Sessions.vue.

I think I'm still misunderstanding you though.

Reply:
Oops I see the confusion. I was looking at this in the "Files changed" tab. This was placed right under where vue data was set, so I thought it was also setting up data. Never mind, this works :)

Reply:
👍 thanks for clarifying. I find the diff to be super confusing sometimes